src/sagemaker_pytorch_serving_container/default_inference_handler.py

```diff
@@ -12,29 +12,39 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
+import os
 import textwrap
 
 import torch
-
 from sagemaker_inference import content_types, decoder, default_inference_handler, encoder
 
+INFERENCE_ACCELERATOR_PRESENT_ENV = "SAGEMAKER_INFERENCE_ACCELERATOR_PRESENT"
+DEFAULT_MODEL_FILENAME = "model.pt"
```

Comment:
nit: be consistent with your quotes. You are using both single and double quotation marks.

Reply:
done, also updated the test file to make quotes consistent